.github/workflows/alerting-swagger-gen.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  gen-swagger:
+    name: Alerting Swagger spec generation cron job
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Set go version
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20.6'
+      - name: Build swagger
+        run: |
+          make -C pkg/services/ngalert/api/tooling post.json
```

Comment:
I wonder why not `all` command?

Reply:
`all` does a bit more than I think makes sense for this PR, but I agree this is too narrow. I'm with @JacobsonMT that it should run `api.json` as well. Fingers crossed they don't differ due to that go-swagger bug...